project_task_pull_request/models/project_task.py

```diff
@@ -10,6 +10,7 @@ class ProjectTask(models.Model):
 
     pr_uri = fields.Char(
         string='PR URI',
+        track_visibility='onchange',
```

Comment:
URLs are properly parsed?

Reply:
No idea, but I don't think that's relevant for this case. :thinking:

Reply:
Well, we want it for being clickable, so not having it it's not very useful

Reply:
Right now this happens:

![Captura de pantalla de 2019-09-17 12-24-44](https://user-images.githubusercontent.com/973709/65037741-3e705c80-d946-11e9-94e4-e5a9fdbfdc17.png)

Yeah it should be clickable :thinking:

Reply:
That's what I suspect, so I'm not sure this is very helpful for what we want.

Reply:
I'm digging through the code and it's not gonna be straightforward to find the place where to fix that.

Could we consider this an improvement and leave the clickability for the future?

Reply:
Well, if you can do both together, we can then change the procedure. Meanwhile, this as is doesn't serve for our purpose.

Reply:
If you recieve the notification via email, though, I think yes it will be clickable, because Gmail does that for you.

Reply:
Also my purpose was more avoiding to write twice the link, that can be done with this as it is, as you can still click it in the field.

Reply:
Yes, but I don't want to need to enter Odoo for clicking to the URL. If Gmail parses it (I'm doubtful about this), then it's enough.